### PR TITLE
data: chiragjh7

### PR DESCRIPTION
**Pull request by chiragjh7**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4654"></a>

### CODE REVIEW

File: data/chiragjh7.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Chirag Jhariya",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Hey, I am a backend developer and can help in yours journey too. ✨",
+  "avatar": "https://github.com/chiragjh7.png",
```
Comment (Panquesito7) [inline data/chiragjh7.json:6]: The `avatar` field is no longer required as of a new update.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by chiragjh7 — COMMENTED** [review]
Thanks for the suggestions!

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

File: data/chiragjh7.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Chirag Jhariya",
+  "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline data/chiragjh7.json:3]: We've recently removed the public stats feature, so `displayStatsPublic` property is no longer needed. 
Please remove this from your json. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉 

<p align="center"></p>

**Pull request merged**